index.md

```diff
@@ -355,7 +353,7 @@ Note: the checker limit is 12 branches.
 
 This error occurs when you have more than three levels of nested blocks in your code.
 This type of nesting is a sign that your function is too complex,
-and should be broken down using helper functions.
+and should be broken down using helper functions or [generators][6].
```

Comment:
I had to rebase on current master to resolve some conflicts.

The original comment was made regarding commit `R0101: Mention generators, since they can be useful in encapsulating some forms of nested code.`